realm/realm-annotations-processor/src/test/resources/some/test/RealmDictionaryModelMixedRequired.java

```diff
@@ -16,20 +16,20 @@
 
 package some.test;
 
-import io.realm.Mixed;
+import io.realm.RealmAny;
 import io.realm.RealmDictionary;
 import io.realm.RealmObject;
 import io.realm.annotations.Required;
 
-public class RealmDictionaryModelMixedRequired extends RealmObject {
+public class RealmDictionaryModelRealmAnyRequired extends RealmObject {
```

Comment:
I think the class got renamed but the file didn't - not that it matters much since it's an asset for the annotation processor, but it should have the same name for consistency

Reply:
Miss it, too many file changes.